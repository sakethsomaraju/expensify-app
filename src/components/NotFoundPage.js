import React from 'react';
import {Link} from 'react-router-dom'

const NotFoundPage =()=>{
    return(
    <div><h1>404!</h1> <Link to="/">go home !</Link>
    </div>)
}
export default NotFoundPage