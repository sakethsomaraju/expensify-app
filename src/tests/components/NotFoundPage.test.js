import React from 'react'
import NotFoundPage from '../../components/NotFoundPage'
import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json'

test('To render the not found page correctly ',()=>{
    const wrapper = shallow(<NotFoundPage />)
    expect(toJSON(wrapper)).toMatchSnapshot()
})