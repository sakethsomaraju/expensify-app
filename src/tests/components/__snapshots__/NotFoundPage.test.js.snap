// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`To render the not found page correctly  1`] = `
<div>
  <h1>
    404!
  </h1>
   
  <Link
    replace={false}
    to="/"
  >
    go home !
  </Link>
</div>
`;
